apps/dashboard/app/apps/ood_files_app.rb

```diff
@@ -20,7 +20,7 @@ def candidate_favorite_paths
   # returns an array of other paths provided as shortcuts to the user
   def favorite_paths
     @favorite_paths ||= candidate_favorite_paths.select do |p|
-      p.remote? || p.path.directory? && p.path.readable? && p.path.executable?
+      AllowlistPolicy.default.permitted?(p.path) && (p.remote? || p.path.directory?) && p.path.readable? && p.path.executable?
```

Comment:
I don't think remote locations adhere to these allowlists because they always check the local filesystem. That is, remote filesystems are always allowed.

So maybe this turns into something like this:
```ruby
if p.remote? 
  true
else 
  AllowlistPolicy.default.permitted?(p.path) && p.path.directory? && p.path.readable? && p.path.executable?
end
```